src/commands/theme.js

```diff
@@ -63,9 +64,15 @@ export default function(program) {
 
           writePackageJsonSync(pkg, dirName);
 
-          return startProcess('npm', ['install', '@shopify/slate-tools', '--save-dev', '--save-exact'], {
-            cwd: root,
-          });
+          if (options.yarn) {
+            return startProcess('yarn', ['add', '@shopify/slate-tools', '--dev', '--exact'], {
+              cwd: root,
+            });
+          } else {
+            return startProcess('npm', ['install', '@shopify/slate-tools', '--save-dev', '--save-exact'], {
+              cwd: root,
+            });
+          }
```

Comment:
Nit: format these functions the same way as in the migrate.js file - either on one line or (like here) two. I don't have a preference.